cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -408,7 +410,7 @@ data:
       buffer_queue_full_action block
       # Set the chunk limit conservatively to avoid exceeding the recommended
       # chunk size of 5MB per write request.
-      buffer_chunk_limit 1M
+      buffer_chunk_limit 512k
```

Comment:
We should apply this to `fluentd-gcp-configmap-old.yaml` as well.

Reply:
I'm not sure that we want to make changes to the existing configmap, why do you want to bundle changes to existing configs?

Reply:
Sync'ed offline. Will leave the old config as-is.